packages/calypso-e2e/src/rest-api-client.ts

```diff
@@ -185,10 +191,10 @@ export class RestAPIClient {
 	 * 	- URL
 	 * 	- site owner
 	 *
-	 * @returns {Promise<AllSitesResponse>} JSON array of sites.
+	 * @returns {Promise<AllDomainsResponse>} JSON array of sites.
```

Comment:
The interface `AllDomainsResponse` is just a renaming of the original interface.